src/install/vm/deployment/previous.mdx

```diff
@@ -0,0 +1,74 @@
+---
+componentType: appInfoConfigOption
+optionType: deployment 
+headingText: Upload previously detected  vulnerabilities through Dependabot Bulk Import 
+---
+
+Import previously detected issues so that New Relic has a complete view of the vulnerabilities detected by Dependabot by performing a bulk import.
+
+## Prerequisites
+
+- A New Relic [license key](/docs/accounts-partnerships/install-new-relic/account-setup/license-key) for the account you want to report data to.
+- A Github Personal Access Token (PAT) with these permissions: `repo AND read:org`
+- The name of your Organization on github.com (for example `newrelic` for github.com/newrelic)
+
+## Bulk import vulnerabilities
+
+Bulk import is initiated with an HTTP POST request. Your GitHub PAT is stored in-memory during import and then discarded. It is never persisted to non-volatile storage.
+
+<Callout variant="caution">
+Running bulk import more than once is not recommended. New Relic will remove duplicate imported data in our vulnerability management UI, but queries to the underlying log data will show duplicate data.
+</Callout>
+
+A bulk import HTTP POST request consists of the following. Replace `<NewRelicIngestKey>`, `<GithubPAT>`, and `<OrganizationName>` with the appropriate values:
+
+### Post:
+
+```
+https://security-ingest-processor.service.newrelic.com/v1/security/import/dependabot
+```
+
+### HEADERS
+
+`Api-Key`: your New Relic Ingest License Key
+`Content-Type`: `application/json`
+
+
+### BODY:
+```
+{
+  "serviceApiKey": "<GithubPAT>",
+  "serviceParams": {
+    "orgName":"<OrganizationName>",
+    "repositories": ["<RepositoryName1","RepositoryName2"] \\ optional
+  }
+}
+```
+
+Example CURL command:
+
+```
+curl --location --request POST 'https://security-ingest-processor.service.newrelic.com/v1/security/import/dependabot' \
+--header 'Api-Key: <NewRelicIngestKey>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "serviceApiKey": "<GithubPAT>",
+    "serviceParams": {
+        "orgName":"<OrganizationName>"
+    }
+}'
+```
+
+### Confirming bulk import activity
+
+When you POST to the v1/security/import/dependabot endpoint, the HTTP response will include a request uuid. For example:
+
+```
+{"success":false,"errorMessage":null,"uuid":"4740e3c8-dbc4-46e6-a4b2-a7fb6f918d20"}
+```
+
+The request guid is included in all Log events written to NRDB from the import job. These events are written in real time as the import job runs. To view the status and output of an import as it runs, use this NRQL query (replacing 4740e3c8-dbc4-46e6-a4b2-a7fb6f918d20 with the uuid returned from your HTTP POST):
```

Comment:
for separating MELT data concepts (e.g., separating logs from events), let's just say 'in all `Log` data written to NRDB...' 'These logs are written...'